code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -840,7 +840,9 @@ void ServerGameState::Tick(fx::ServerInstanceBase* instance)
 					float distSquared = (diffX * diffX) + (diffY * diffY);
 					if (distSquared < entity->GetDistanceCullingRadius())
 					{
-						isRelevant = true;
+						if (playerEntity->playerCullingRadius == 0.0f || distSquared < playerEntity->playerCullingRadius) {
```

Comment:
It would be better to include this condition with the one above.
And read the value from entityClient/client data not the entity